aws/aws.go

```diff
@@ -20,6 +20,7 @@ const (
 	reaperTag           = "REAPER"
 	reaperTagSeparator  = "|"
 	reaperTagTimeFormat = "2006-01-02 03:04PM MST"
+	scalerTag           = "Reaper-Autoscaler"
```

Comment:
It would be kind of obnoxious to uppercase this tag, but the other Reaper tags are uppercase so we probably should for consistency.